assets/css/portfolio.css

```diff
@@ -1,3 +1,117 @@
-body{
-    background-color: #fff;
-}
\ No newline at end of file
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap");
+
+/* colors */
+:root {
+  --logo-color: #282e2e;
+  --bold-font-color: #282e2e;
+  --main-font-color: #707070;
+  --sublinks: #707070;
+  --secondary-font-color: #9a9a9a;
+  --leading-font-color: #30ba8a;
+}
+
+/* Global reset */
+* {
+  margin: 0;
+  padding: 0;
+  outline: none;
+}
+a {
+  text-decoration: none !important;
+}
+/* font set */
+
+body {
+  background-color: #fff;
+  font-family: "Poppins", sans-serif;
+}
+header {
+  width: 100%;
+  height: 100vh;
+  background-image: url(/assets/images/girlface.png);
+  background-repeat: no-repeat;
+  background-size: 400px;
+  background-position: right bottom;
+}
+header .navbar {
+  padding: 0;
+}
+header .navbar.navbar-light .navbar-brand {
+  font-size: 35px;
+  font-weight: 800;
+  color: var(--logo-color);
+}
+header .navbar #collapsibleNavbar .nav-item {
+  margin-right: 30px;
+  height: 50px;
+}
+header .navbar #collapsibleNavbar .nav-item .nav-link {
+  color: var(--sublinks);
+  font-size: 14px;
+}
+header .navbar #collapsibleNavbar .nav-item.active {
+  border-top: 8px solid var(--leading-font-color);
+  color: var(--leading-font-color);
+}
+
+/* into content */
+.hero {
+  height: 90vh;
```

Comment:
The page should fit on the screen and look good on 1440px by default. It does not. Please find a work around for this.